packages/lib32-rust-bin/lib32-rust-bin.pacscript

```diff
@@ -0,0 +1,24 @@
+pkgname="lib32-rust-bin"
+gives="lib32-rust"
+pkgver="1.80.1"
+pkgdesc="Rust is a systems programming language that runs blazingly fast, prevents segfaults, and guarantees thread safety"
+arch=("x86_64" "aarch64")
+repology=("project: rust")
+url='https://www.rust-lang.org/'
+source=("https://static.rust-lang.org/dist/rust-${pkgver}-i686-unknown-linux-gnu.tar.xz")
+sha256sums=("9af695dd4eacb397c2169abdfae35c6342f210d04632571d5faf924fa2cc6d39")
+provides=(libstd-rust-dev:i386)
```

Comment:
```suggestion
provides=("libstd-rust-dev:i386")
```

Reply:
> ```suggestion
> provides=("libstd-rust-dev:i386")
> ```

I need to verify this. Because it's amd64 crossbuild package. Not sure about the provides :/